Packs/Legacy/Scripts/GetMLModelEvaluation/GetMLModelEvaluation_test.py

```diff
@@ -1,7 +1,5 @@
 import json
 
```

Comment:
please add some more edge cases, like when y_true != y_pred in any case, or y_true == y_pred always